linera-sdk/src/test/validator.rs

```diff
@@ -67,7 +67,7 @@ impl Clone for TestValidator {
 impl TestValidator {
     /// Creates a new [`TestValidator`].
     pub async fn new() -> Self {
-        let key_pair = SigningKey::generate();
```

Comment:
Shouldn't this be a validator key?

Reply:
Done in [Use proper keys in tests](https://github.com/linera-io/linera-protocol/pull/3366/commits/70fe86eb4bed9152d1526f990edcf99735f48c97)